apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html

```diff
@@ -11,26 +11,26 @@
             <div class="col-md-2 col-sm-6 mb-2">
                 <sh-select [options]="selectYear" [fixedValue]="activeFinancialYearLabel" [placeholder]="'Select Year'" name="financialYear" (selected)="selectFinancialYear($event);" [forceClearReactive]="forceClear$ | async"></sh-select>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
+            <div class="col-md-2 col-sm-6 mb-2 clear-both">
                 <div class="col-md-6 col-sm-6 pl-0">
                     <sh-select [options]="fromMonthNames" [placeholder]="'From Month'" name="fromMonth" [(ngModel)]="fromMonth" [forceClearReactive]="forceClear$ | async" (selected)="selectFromMonth($event);"></sh-select>
                 </div>
                 <div class="col-md-6 col-sm-6 pr-0">
                     <sh-select [options]="toMonthNames" [placeholder]="'To Month'" name="toMonth" [(ngModel)]="toMonth" [forceClearReactive]="forceClear$ | async" (selected)="selectToMonth($event);"></sh-select>
                 </div>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
+            <div class="col-md-2 col-sm-6 mb-2 clear-both">
                 <sh-select [options]="flatGroupsOptions" [placeholder]="'Group *'" name="group"
                     (selected)="selectGroup($event);" [forceClearReactive]="forceClear$ | async"></sh-select>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
+            <div class="col-md-2 col-sm-6 mb-2 clear-both">
                 <input type="checkbox" name="showHideOpeningClosingBalance" id="showHideOpeningClosingBalance" [(ngModel)]="exportRequest.showHideOpeningClosingBalance" [value]="true">&nbsp;<label for="showHideOpeningClosingBalance">Opening/Closing Balance</label>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
+            <div class="col-md-2 col-sm-6 mb-2 clear-both">
                 <input type="checkbox" name="balanceTypeAsSign" id="balanceTypeAsSign" [(ngModel)]="exportRequest.balanceTypeAsSign" [value]="true">&nbsp;<label for="balanceTypeAsSign">Cr/Dr with +/-</label>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
-                <button class="btn btn-pink w127" [disabled]="!isFormValid() || isLoading" (click)="exportReport();">Export</button>
```

Comment:
why this class is removed?

Reply:
i think no need to add width, we follow default width

Reply:
Meghna asked me to increase width of button that's why i added that class. You should confirm the use before removing.

Reply:
meghna confirmed to remove class